README.md

```diff
@@ -34,6 +34,7 @@ fn main() {
                     .skip_while(|l| !l.starts_with("//"))
                     // Extract consecutive commented lines.
                     .take_while(|l| l.starts_with("//"))
+                    // Strip the initial "//" from commented lines.
```

Comment:
Why don't we define `//` as a `const LANG_TEST_COMMENT`, replace the duplicated strings above, and then do `.map(|l| &l[LANG_TEST_COMMENT.len()..])`?

This way the comment can be changed in one place.

Reply:
I'm not sure that makes things clearer TBH.

Reply:
My plunder earlier is evidence that it would be better :)

To change the comment here, I'd have to update 3 places, one of which can't be detected by searching for the comment string.

Reply:
"Plunder" is a wonderfully Freudian typo ;)

I take your point though. Fixed in https://github.com/softdevteam/lang_tester/pull/79/commits/109f597f923a95c53e09501838f1d35533f366a3.